### ISSUE TITLE
Variables don't working

### ISSUE THREAD

**Issue by fabionaspolini** [aelassas/Wexflow#130]
I'm trying to use local variables at FilesLoader and FtpTask but dont working.
I'm use dotnet core on linux platform, latested version 4.8.

My confile file looks like this: 

```
<Workflow id="1" name="Upload database backup" description="Upload database backup" xmlns="urn:wexflow-schema">
  <Settings>
    <Setting name="launchType" value="trigger" />
    <Setting name="enabled" value="true" />
    <Setting name="approval" value="false" />
  </Settings>
  <LocalVariables>
    <Variable name="folder" value="/opt/wexflow/MyTestsWorkflow/UploadBackupTask" /> 
    <Variable name="ftp" value="myaddress.com" /> 
  </LocalVariables>
  <Tasks>
   ...
    <Task id="3" name="FilesLoader" description="Carregar arquivo de backup" enabled="true">
      <Setting name="folder" **value="$folder"** />
    </Task>
    .........
    <Task id="6" name="Ftp" description="Listar arquivos FTP" enabled="true">
      <Setting name="command" value="list" />
      <Setting name="protocol" value="ftp" />
      <Setting name="server" **value="$ftp"** />
      <Setting name="port" value="21" />
      <Setting name="user" value="xxxx" />
      <Setting name="password" value="xxxxx" />
      <Setting name="path" value="/Teste/" />
    </Task>
   ......
</Workflow>
```

I looked at the source code, and realized that the _Workflow.Parse_ method is no longer running, it was commented recently.

**Comment by aelassas** [comment]
@fabionaspolini Yes, local variables and global variables are no longer working since storing workflows in the database. That's why I commented Workflow.Parse method. I will work on that soon.

**Comment by aelassas** [comment]
@fabionaspolini I added local variables and global variables in the upcoming version 4.9. I will notify you once I release the v4.9.

**Comment by aelassas** [comment]
@fabionaspolini I've just released the [v4.9](https://github.com/aelassas/Wexflow/releases/tag/v4.9).
